node/pkg/wfetcher/providers/coinbase/utils.go

```diff
@@ -0,0 +1,37 @@
+package coinbase
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"bisonai.com/orakl/node/pkg/wfetcher/common"
+	"github.com/rs/zerolog/log"
+)
+
+func TickerToFeedData(ticker Ticker, feedMap map[string]int32) (*common.FeedData, error) {
+	feedData := new(common.FeedData)
+
+	timestamp, err := time.Parse(time.RFC3339Nano, ticker.Time)
```

Comment:
Do they really provide time in nanoseconds? 🤔

Reply:
```
2022-10-19T23:28:22.061769Z
```
this is example output, I thought it was RFC3339Nano type?

Reply:
Interesting, they provide nanoseconds 🔥